### PR TITLE
Addition of Aria attributes and removal of unnecessary links

### PR DESCRIPTION
**Pull request by KudosAbhay**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,10 +29,10 @@ <h1>Mercury Logo</h1>
                 </div>
                 <div class="member-navigation">
                     <ul>
-                        <li><a href="https://t1-www.cxtrvl.com/BaseResponsive" data-qaid="Link_RewardsHome">Rewards Home</a></li>|
-                        <li><a href="https://t1-www.cxtrvl.com/BaseResponsive/MyProfile" data-qaid="Link_MyProfile">Profile</a></li>|
-                        <li><a href="https://t1-www.cxtrvl.com/BaseResponsive/MyTrips" data-qaid="Link_MyTrips">My Trips</a></li>|
-                        <li><a href="#" class="js-sign-out" data-qaid="Link_SignOut">Sign Out</a></li>
+                        <li><a href="#">Rewards Home</a></li>
```
Comment (prateek3189) [inline index.html]: instead of  **_#_** put **_javascript:void(0)_**

File: index.html [hunk 2]
```diff
@@ -56,20 +56,18 @@ <h1>Mercury Logo</h1>
 
         <div class="offers-div" data-ad-sct="0">
             <div class="banner">
-                <div class="marketing-text-banner">
-                    <div class="text">
-                        <h2>Spread your wings and discover new destinations!</h2>
+                <div class="marketing-text-banner" aria-labelledby="offers-heading">
+                    <div class="text" >
```
Comment (prateek3189) [inline index.html]: Remove extra space here

**Comment by KudosAbhay** [comment]
Okay. Done!

**Comment by KudosAbhay** [comment]
Okay. Done! Please check.

**Pull request merged**